src/components/utils/MenuItem/MenuItem.tsx

```diff
@@ -9,6 +9,9 @@ interface MenuItemProps extends NativeAttributes<'div'> {
   /** Whether the particular item should be disabled */
   disabled?: boolean;
 
+  /** Whether the item is nested in a group or not */
+  grouped?: boolean;
```

Comment:
rename to `nested`? I think it makes more sense in the context of this component

```suggestion
  nested?: boolean;
```